File: Sentinel/iotjava/iotrmi/Java/basics/TestClass_Skeleton.java
import java.net.InetAddress;

public class TestClass_Skeleton {

	public static void main(String[] args) throws Exception {

		int portsend = 5000;
		int portrecv = 6000;
		String callbackAddress = InetAddress.getLocalHost().getHostAddress();
		//TestClassProfiling tc = new TestClassProfiling();
		TestClass tc = new TestClass();
		TestClassInterface_Skeleton tcSkel = new TestClassInterface_Skeleton(tc, portsend, portrecv);
	}
}

File: Sentinel/iotjava/iotrmi/C++/basics/TestClass_Skeleton.cpp
#include <iostream>
#include <string>
#include "TestClassInterface_Skeleton.hpp"
#include "TestClass.hpp"

using namespace std;

int main(int argc, char *argv[])
{
	TestClassInterface *tc;
	TestClassInterface_Skeleton *tcSkel;

	int port = 5010;
	//tc = new TestClass(3, 5.0, "7911");
	tc = new TestClassProfiling();
	string callbackAddress = "128.195.204.132";
	tcSkel = new TestClassInterface_Skeleton(tc, callbackAddress, port);

	delete tc;
	delete tcSkel;
	return 0;
}
